Shared/Actions/AssetsUpload/ArchiveResult.cs

```diff
@@ -114,7 +114,12 @@ internal override void SetValues(JToken source)
             Placeholder = source.Value<bool>("placeholder");
             AccessMode = source.Value<string>("access_mode");
             ResourceCount = source.Value<int>("resource_count");
-            Tags = source["tags"].ToObject<string[]>();
+
+            var tags = source["tags"];
```

Comment:
This file should use annotations, like 
`[DataMember(Name = "resource_type")]`, etc.

and method `SetValues` should be removed. 

Please refactor.